styles/signals.mapcss

```diff
@@ -617,9 +630,47 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:minor"="DE
 	allow-overlap: true;
 }
 
-/********************************************/
+/*******************************************/
+/* FI shunting light signals type Ro (new) */
+/*******************************************/
+node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:shunting"="FI:Ro"]["railway:signal:shunting:states"="FI:Ro0;FI:Ro1;FI:Ro2"]["railway:signal:shunting:form"=light]
```

Comment:
This enforces a specific ordering of the signal states, which we do not otherwise enforce because basically any order can happen. If exactly these 2 states must be present and nothing else then you should probably add another rule that checks for the opposite order, if additional states are permitted then you should use ~= and should check for both of them separately.